applications/osb-portal/src/components/menu/MainMenu.tsx

```diff
@@ -16,29 +17,58 @@ const useStyles = makeStyles(() => ({
   firstButton: {
     fontWeight: 600,
   },
+  flipButton: {
+    fontWeight: 700,
+    textTransform: "uppercase",
+    padding: '5px',
+    fontSize: '0.75rem',
+  },
 }));
 
 export const MainMenu = () => {
   const classes = useStyles();
   const history = useHistory();
   return (
-    <Box display="flex" flexWrap="wrap" p={0} bgcolor="background.paper">
-
-      <MainMenuItem
-        title="OSB"
-        className={classes.button + " " + classes.firstButton}
-        items={[
-          { label: "About", callback: () => alert("Open Source Brain v2") },
-        ]}
-      />
-      <MainMenuItem
-        title="View"
-        className={classes.button}
-        items={[
-          { label: "Repositories", callback: () => history.push("/repositories") },
-          { label: "Workspaces", callback: () => history.push("/") },
-        ]}
-      />
+    <Box display="flex" flexWrap="wrap" p={0} bgcolor="background.paper" justifyContent="space-between">
+      <Box display="flex" flexWrap="wrap" p={0}>
+        <MainMenuItem
+          title="OSB"
+          className={classes.button + " " + classes.firstButton}
+          items={[
+            { label: "About", callback: () => alert("Open Source Brain v2") },
+          ]}
+        />
+        <MainMenuItem
+          title="View"
+          className={classes.button}
+          items={[
+            { label: "Repositories", callback: () => history.push("/repositories") },
+            { label: "Workspaces", callback: () => history.push("/") },
+          ]}
+        />
+      </Box>
+      {
+        history.location.pathname === "/" ?
+        <MainMenuItem
+          title={<>WORKSPACES <ExpandMoreIcon fontSize="small"/></>}
```

Comment:
Minor: when the ui is uppercase better write the word in lowecase and use text transform